doc.go

```diff
@@ -112,6 +112,8 @@ The following connection parameters are supported:
 	* validateDefaultParameters: true by default. Set to false to disable checks on existence and privileges check for
 								 Database, Schema, Warehouse and Role when setting up the connection
 
+    * tracing: Specifies the logging level to be used. Set to error by default.
```

Comment:
What other levels does the parameter support?

Reply:
Updated to include valid values.